dolphin/drivers/manager.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oslo_log import log
+
+
+LOG = log.getLogger(__name__)
+
+DRIVER_MAPPING = {
+    "fake_storage": "dolphin.drivers.fake_storage.FakeStorageDriver"
+}
+
+
+class DriverManager(object):
+
```

Comment:
Should we have interface for driver to register? how drivers are inititilized? and how DriverManger select the driver ?

Reply:
yes, it should support, I didn't expose internal behavior in this pr. The client who uses the driver manager doesn't care about them.

Reply:
ok fine..